cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -64,9 +65,18 @@ spec:
           - name: grafana-persistent-storage
             mountPath: /var
       volumes:
-      - name: influxdb-persistent-storage
-        persistentVolumeClaim:
-          claimName: influxdb-claim
       - name: grafana-persistent-storage
         emptyDir: {}
-  serviceName: monitoring-influxdb
+  volumeClaimTemplates:
+  - metadata:
+      name: influxdb-ps
+      namespace: kube-system
```

Comment:
you shoulnd't need the namespace in here the pvc should get created in the same namespace